docs/content/intro/community-roles.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Participating in the Athens Community"
+date: 2018-02-11T15:57:56-05:00
+---
+
+Absolutely everyone is welcome to join our community at any time! We
+are a friendly and inclusive group and we'd love to have you. We have three roles in the Athens community:
+
+- Community member
+- Contributor
+- Maintainer (we also call this core maintainer sometimes. They're the same thing)
```

Comment:
nit: I'd move this parenthetical bit to the maintainer section itself, just because it sticks out here.

Reply:
:+1: